### PR TITLE
[MK-759] Interactive tooltip

### PR DESCRIPTION
**Pull request by Rheed**
Ping!

@cvpinhe

### CODE REVIEW

**Comment by cvpinhe** [comment]
@carlos-dev

File: tooltip.coffee [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
+			content: '' # Tooltip content
+			trigger: 'hover' # How tooltip is triggered (hover or manual)
+			style: '' # Apply a css class
+			width: false # Limit content width
+			tip: true # show tip (true or false)
+			delay: 0 # delay to show tooltip
+			border: false # if true, duplicates the tip
+			selector: false # if is defined, events are attached at selector
+			interactive: false # if the content can be interactive with the user
+			onShow: null
```
Comment (cvpinhe) [inline tooltip.coffee]: Missing comment here explaining what onShow is.

File: tooltip.coffee [hunk 2]
```diff
@@ -22,11 +21,12 @@
 		topWindow = self
 
 		tS = true
+		hideDelay = 500
```
Comment (cvpinhe) [inline tooltip.coffee]: Look we have delay to show the tooltip... I think you can set the hideDelay like that and when you want to use that you can set it as a parameter

Comment (Rheed) [inline tooltip.coffee]: Yeah noticed this when doing the PR; will add these to the platform too :D

File: tooltip.coffee (hunk 1 above)
Comment (davesnx) [inline tooltip.coffee]: sync align for the comments? :heart:

File: tooltip.coffee [hunk 3]
```diff
@@ -156,17 +174,20 @@
 				when "right"
 					top = elT - (tipH - elH) / 2
 					left = elL + elW
+				#The magic number 5 is an offset
```
Comment (davesnx) [inline tooltip.coffee]: @Rheed are you sure that doesn't affect at the actual popup's?

File: tooltip.coffee (hunk 1 above)
Comment (Rheed) [inline tooltip.coffee]: Sure :grin: :+1:

**Comment by Rheed** [comment]
@cvpinhe @carlos-dev gonna close this, as it's probably that will receive more changes when adapting it to the other pages, will create a new PR with the final changes

**Pull request closed without merging**